### ISSUE TITLE
Add landcover in percent to API

### ISSUE THREAD

**Issue by patrick-zippenfenig** [open-meteo/open-meteo#108]
Suggested by @Wxglobal

See https://github.com/open-meteo/open-meteo/issues/103#issuecomment-1214302980

**Comment by Wxglobal** [comment]
Hi Patrick i think if landcover fraction will be added to add the model elevation too in order to better trim the grid forecast with the reference st the location elevation giveng by the elevation Api

**Comment by drmrbrewer** [comment]
@patrick-zippenfenig I note that there is a **landsfc** parameter in the GFS/HRRR data, at least from [this source](https://nomads.ncep.noaa.gov/dods/gfs_0p25_1hr/gfs20230603/gfs_0p25_1hr_00z.info) for the GFS data: "surface land cover (0=sea, 1=land) [proportion]"... and indeed a whole load of other stuff... is this "landsfc" parameter available in the GRIB data you're using?

**Comment by patrick-zippenfenig** [comment]
Yes, it to use land cover fraction from GFS/HRRR. Currently, I only mask-out locations on sea, but do not store the actual land cover fraction. I will need to refactor the entire code to store all static attributes on just one file for fast file access. There is also additional information like soil and vegetation type, that can be included.

**Comment by Wxglobal** [comment]
Yes is this

Il Lun 5 Giu 2023, 12:18 Patrick Zippenfenig ***@***.***> ha
scritto:

**Comment by Wxglobal** [comment]
Type of soil and vegetation index are useful for forecasting dowscaling on
local basis using ML

Il Lun 5 Giu 2023, 14:00 Paolo Caraccio ***@***.***> ha scritto:
